lab-01-Jacob/package.json

```diff
@@ -13,6 +13,11 @@
   "license": "ISC",
   "devDependencies": {
     "chai": "^3.5.0",
+    "eslint": "^3.10.2",
+    "eslint-config-google": "^0.7.0",
```

Comment:
Travis-CI is showing your test as failing because it's referencing the repo's package.json, but looking good here!